src/style.css

```diff
@@ -0,0 +1,34 @@
+* {
+  margin: 0;
+}
+
+.stars-container {
```

Comment:
container should not have any sizes set, let the content make its size

Reply:
i did it, but now i have 0 test completed caused by some invisible margin/top.
Check it out  😄

Reply:
I'm not really sure, but I bet that's because you added `justify-content`, so the tests now see it differently. I don't see anything critical here though. Looks good 👍